bank/sql_bank/README.md

```diff
@@ -33,5 +22,5 @@ Run against an existing cockroach node or cluster.
 # Launch your node or cluster in secure mode with certificates in [mycertsdir]
 # Find a reachable address:[mycockroach:26257].
 # Run the example with:
-./sql_bank https://mycockroach:26257/?certs=mycertsdir
+./sql_bank "postgres://root@mycockroach:26257?sslcert=mycertsdir/root.client.crt&sslkey=mycertsdir/root.client.key"
```

Comment:
disagree.

in fact, my actual command line _was_ using expansion for `~/.../root.client.key` and it isn't unreasonable to expect others to as well.